docs/vocascan-server/customPages.md

```diff
@@ -0,0 +1,131 @@
+# Custom Pages
+
+We provide you with the possibility to host your own static pages or make redirects to other websites on your Vocascan
+server and make them accessible to the public. This can be very helpful if you have to comply with legal requirements
+and therefore have to provide a privacy policy or other information. To add your page to the server, you need to add the
+files as a volume to the Docker container and define them in `vocascan.config.js` or the `env variables`.
+
+!> The Vocascan frontend needs predefined routes (`url`) to find specific pages:</br> 1. Legal Notice:
```

Comment:
what about adding that as a table so if we later add some more?

Reply:
I think you can't add a table to a `!>` tag

Reply:
I think that's not possible, but there is no need for a !> block. Maybe just use a separate headline where you describe for what these paths are.